### PR TITLE
Changing assembly and package name to have the word 'Plugin'

### PR DESCRIPTION
**Pull request by nemakam**
## Description
Changing assembly and package name to have the word `Plugin`.
`Microsoft.Azure.ServiceBus.Keyvault` can be very misleading

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] **I have read the [contribution guidelines](./CONTRIBUTING.md).**
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR.
- [ ] The pull request does not introduce breaking changes (unless a major version change occurs in the assembly and module).
- [ ] If applicable, the public code is properly documented.
- [ ] Pull request includes test coverage for the included changes.
- [ ] The code builds without any errors.

### CODE REVIEW

**Review by SeanFeldman — CHANGES_REQUESTED** [review]
No a fan of having both namespace and a class with the same name, especially when the class is in the public API. Can I suggest to make the namespace something like `Microsoft.Azure.ServiceBus.Plugins`? For discoverability, if the namespace is `Microsoft.Azure.ServiceBus`, anyone adding this package will not have to "discover" where those types are.

**Comment by nemakam** [comment]
> For discoverability, if the namespace is Microsoft.Azure.ServiceBus, anyone adding this package will not have to "discover" where those types are.

@SeanFeldman Not sure what you mean here... We will be having separate packages for each of the plugin. That was the idea behind different namespaces.

**Comment by SeanFeldman** [comment]
@nemakam doesn't have to be a separate namespace. Namespace is for partitioning when that one is needed. You could safely have a Microsoft.Azure.ServiceBus namespace in 2 packages that are related. That way, as a user, by including the package, I don't need to look for the right namespace.

File: README.md [hunk 1]
```diff
@@ -18,5 +18,5 @@ See our [Contribution Guidelines](./.github/CONTRIBUTING.md).
 
 The folling plugins are included with this repository:
 
-* [Key Vault](./docs/keyvault.md)
-* [Message ID](./docs/messageid.md)
\ No newline at end of file
+* [Key Vault](./src/Microsoft.Azure.ServiceBus.KeyVault/readme.md)
```
Comment (vinaysurya) [inline README.md]: >KeyVault [](start = 47, length = 8)

KeyVaultPlugin and MessageIdPlugIn?

**Comment by vinaysurya** [comment]
:shipit:

**Review by vinaysurya — APPROVED** [review]

**Pull request merged**